dht.go

```diff
@@ -512,6 +512,10 @@ func (d *DHT) needMoreNodes() bool {
 	return n < minNodes || n*2 < d.config.MaxNodes
 }
 
+func (d *DHT) GetNumNodes() int {
+        return d.routingTable.numNodes()
```

Comment:
Do we really need this? I don't think we do and I think it's better to only expose methods when we really need to. Besides, I think this isn't safe to be used concurrently while the DHT is running?

Reply:
agree. was just usefull for my debuging.